code/API_definitions/qos-booking-and-assignment.yaml

```
@@ -1652,7 +1652,11 @@ components:
             longitude: -127.10066
           radius: 100
         sink: "https://application-server.com/notifications"
-        sinkCredential: {}
+        sinkCredential:
+          credentialType: ACCESSTOKEN
+          accessToken: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."

```

To avoid alarms from SecurityScanner I would shorten it even further or completely replace with a placeholder:

```suggestion
          accessToken: "<access_token>"
```